sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -67,6 +67,11 @@
                 offline_store_creator=BigQueryDataSourceCreator,
                 online_store=REDIS_CONFIG,
             ),
+            IntegrationTestRepoConfig(
```

Comment:
This is a list of configurations for all tests. So this will add an additional option to parametrized tests including tests that work with an online store.
So probably it's not what we want. Adding tests to `test_cli` and `test_repo_config` should be enough